utils/kernel.c

```diff
@@ -127,6 +127,9 @@ bool check_kernel_pid_filter(void)
 	bool ret = true;
 	char *filename = get_tracing_file("options/function-fork");
 
+	if (filename == NULL)
+		return false;
```

Comment:
I'm not sure if it could return NULL.

Reply:
Would it be better to remove that null check?

Reply:
No, I think it's possible.  Please keep it.

Reply:
I got it. ;)